### DISCUSSION TITLE
Turn by Turn Navigation Support

### DISCUSSION THREAD

**Discussion by kosidinma** [liodali/osm_flutter#145] [Q&A]
First of all, I'd like to say great work on this package.
This has probably been asked before but I was wondering if this supports turn by turn navigation or if there are plans to.
Unfortunately didn't see anything in the documentation regarding this.

If this already exists, please do you have any examples??

Thanks!

**Comment by liodali** [comment]
you can implemented manually by combining several function
for example you limit navigation in the map with `limitAreaMap(BoundingBox(e:maxLogitude,n:maxLatitude,w:minLog,s:minLat))`
you activate trackLocation and zoomIn to userLocation
but i will add also instruction of the route that you will used to show more information about the trip
this feature will be added after i will make this library stable to implement such feature

**Reply by kosidinma** [reply]
Thank you 🙂🙂
